R/use_python.R

```diff
@@ -87,7 +87,33 @@ use_condaenv <- function(condaenv = NULL, conda = "auto", required = FALSE) {
     stop("Unable to locate conda environment '", condaenv, "'.")
 
   if (!is.null(condaenv))
-    use_python(conda_env_python, required = required)
+    use_python(conda_env_python[[1]], required = required)
```

Comment:
One thing that surprised me: even if you supply a Conda binary explicitly, the different Conda installations seem to know about each other and their environments. This means that `conda_list()` will return environments from _all_ Conda installations discovered on the system.

I wonder if we need to limit the check here to Conda environments explicitly associated with / contained within the associated Conda installation?